roster2/src/handlebars/members.handlebars

```diff
@@ -66,7 +66,7 @@
                 {{/if}}
             {{/if}}
 
-            {{#if hasGroups}}
+            {{#if ../viewGroup}}
```

Comment:
The if statement cannot handle conditionals.

If the members loaded has no groups associated with them, but there are groups associated with other unloaded members than the groups header will not be shown (this was present with the hasGroups condition).